knowrob_objects/prolog/knowrob_objects.pl

```diff
@@ -34,11 +34,13 @@
       storagePlaceFor/2,
       storagePlaceForBecause/3,
       current_object_pose/2,
+      new_current_object_pose/2,
```

Comment:
Don't introduce new predicates. You can define rules with identical names but different
bodies.
Then, `current_object_pose` would support old-style and new-style